webapp/javascript/components/FlameGraph/FlameGraphComponent/index.tsx

```diff
@@ -139,7 +142,18 @@ export default function FlameGraphComponent(props: FlamegraphProps) {
           ExportData={ExportData}
         />
 
-        <div>
+        {dataUnavailable ? (
+          <div className="error-message">
```

Comment:
If this class is used only here, would you mind using css modules? We are now trying to scope the css as much as possible.

Just to give you some direction, you have to create a `index.css.module`, for example
https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/components/FlameGraph/FlameGraphComponent/Highlight.tsx#L2
https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/components/FlameGraph/FlameGraphComponent/Highlight.module.css

Reply:
Done!